shared/src/commonMain/composeResources/values-fr/strings.xml

```diff
@@ -44,4 +44,6 @@
     <string name="no_paired_watch">Pas de montre appairée</string>
     <string name="no_bookmarks">Aucun favori défini</string>
     <string name="privacy_policy">Politique de confidentialité</string>
+    <string name="bookmarks_past">Passé</string>
+    <string name="bookmarks_upcoming">À venir</string>
```

Comment:
no idea if these are correct, feel free to change them

Reply:
@martinbonnin wdyt? We can follow up with another PR if they do need to be changed